napari/components/_viewer/view/view.py

```diff
@@ -16,10 +17,12 @@ class QtViewer(QSplitter):
     def __init__(self, viewer):
         super().__init__()
 
-        QCoreApplication.setAttribute(Qt.AA_UseStyleSheetPropagationInWidgetStyles, True)
+        atribute = Qt.AA_UseStyleSheetPropagationInWidgetStyles
```

Comment:
why? also you misspelled "attribute"

Reply:
the line was so long it was a pep8 error - i have fixed the spelling

Reply:
would look into using one of the following options instead then:
```python
QCoreApplication.setAttribute(Qt.AA_UseStyleSheetPropagationInWidgetStyles, 
                              True)
```

```python
QCoreApplication.setAttribute(
    Qt.AA_UseStyleSheetPropagationInWidgetStyles, True
)
```

```python
QCoreApplication.setAttribute(
    Qt.AA_UseStyleSheetPropagationInWidgetStyles,
    True
)
```

Reply:
The middle-one works fine, I am going with that one